wbb/modules/images.py

```diff
@@ -0,0 +1,63 @@
+from wbb.utils import cust_filter
+from wbb import app, WALL_API_KEY
+import urllib.request
+import json
+from random import randint
+import requests as r
+
+__MODULE__ = "Images"
+__HELP__ = '''/cat - Get Cute Cats Images
+/dog - Get Cute Dogs Images
+'''
```

Comment:
I think it will look better if you just use `\n`
```
__HELP__ = "/cat - Get Cute Cats Images\n" +
           "/dog - Get Cute Dogs Images"
```
(not sure about indent here, check by yourself)

Reply:
done